src/azul/plugins/metadata/hca/contributor_matrices.py

```diff
@@ -112,7 +112,8 @@ def make_stratification_tree(files: Sequence[Mapping[str, str]]) -> JSON:
                                     "name": "n",
                                     "size": 1,
                                     "source": "s",
-                                    "url": null
+                                    "url": null,
+                                    "is_phantom": false
```

Comment:
It's not immediately clear to me why we need this additional property. Please explain in PL.